File: src/01-easy/the-descent/solution.pl
while (1) {
    my ($max, $idx) = (-1, 0);
    for my $i (0..7) {
        chomp(my $h = <STDIN>);
        if ($h > $max) { $max = $h; $idx = $i; }
    }
    print "$idx\n";
}

File: src/01-easy/the-descent/solution.php
<?php
while (true) {
    $max = -1; $idx = 0;
    for ($i = 0; $i < 8; $i++) {
        $h = intval(trim(fgets(STDIN)));
        if ($h > $max) { $max = $h; $idx = $i; }
    }
    echo $idx . "\n";
}
